services/ordering/src/EShopOnAbp.OrderingService.HttpApi.Host/EShopOnAbp.OrderingService.HttpApi.Host.csproj

```diff
@@ -10,6 +10,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="DistributedLock.Redis" Version="1.0.1" />
```

Comment:
`DistributedLock.Redis` is already added to microservice shared project.

Reply:
Okay, agreed. I had focused on the code and still need to refactor it. Thanks for the feedback.